### ISSUE TITLE
[SP2019]Make BOM for labs 

### ISSUE THREAD

**Issue by trickeydan** [sourcebots/tasks#47]

**Comment by jcwilliamson** [comment]
How much stuff are we allowed/willing to take from existing lab stock in 59/16

**Comment by trickeydan** [comment]
cc Tyler

**Comment by trickeydan** [comment]
Blocked on #46

**Comment by jcwilliamson** [comment]
I think before much more progress can be made on this, we should finish #25 so we can put some stuff specifically aimed at the game.

**Comment by joe9599** [comment]
The BOM will exist here https://docs.google.com/spreadsheets/d/1nahSkuvf5Tf4m66dcgQeNHO71B6hP9l7xcWeof3eRZI/edit#gid=0

**Comment by Tyler-Ward** [comment]
Bom has been merged into the Order5 purchaces
